tests/farmers.test.js

```diff
@@ -81,36 +82,46 @@ describe('Farmer route', () => {
           done(err);
         });
     });
-    it('It deletes farmer details', (done) => {
+    // it('It deletes farmer details', (done) => {
+    //   chai
+    //     .request(server)
+    //     .delete(`/api/v1/farmers/${id}/delete`)
+    //     .set('Authorization', token)
+    //     .end((err, res) => {
+    //       res.should.have.status(200);
+    //       res.body.message.should.equal('Farmer details deleted successfully');
+    //       done(err);
+    //     });
+    // });
+    it('It should return an array of farmers', (done) => {
       chai
         .request(server)
-        .delete(`/api/v1/farmers/${id}/delete`)
+        .get('/api/v1/farmers')
         .set('Authorization', token)
         .end((err, res) => {
-          res.should.have.status(200);
-          res.body.message.should.equal('Farmer details deleted successfully');
+          res.should.have.status(404);
           done(err);
         });
     });
-    it('It should return an array of farmers', (done) => {
+    it('It should return 201', (done) => {
       chai
         .request(server)
-        .get('/api/v1/farmers')
+        .post('/api/v1/farmers/create')
         .set('Authorization', token)
+        .send(farmerInput)
         .end((err, res) => {
-          res.should.have.status(404);
+          res.should.have.status(201);
           done(err);
         });
     });
-    it('It should return 201', (done) => {
+    it('It should return 409', (done) => {
```

Comment:
❓What are these test changes?

Reply:
It's to test if the farmer exists before. So, instead of getting 201 after registering the same farmer again, you get 409